mappings/net/minecraft/world/biome/DefaultBiomeFeatures.mapping

```diff
@@ -64,10 +81,16 @@ CLASS net/minecraft/class_3864 net/minecraft/world/biome/DefaultBiomeFeatures
 	FIELD field_21180 RED_MUSHROOM_BLOCK Lnet/minecraft/class_2680;
 	FIELD field_21181 BROWN_MUSHROOM_BLOCK Lnet/minecraft/class_2680;
 	FIELD field_21182 MUSHROOM_BLOCK Lnet/minecraft/class_2680;
+	FIELD field_21183 JUNGLE_TREE_2_CONFIG Lnet/minecraft/class_4640;
```

Comment:
more descriptive name plz? is this for large jungle trees or something?

Reply:
it has no cocoa beans iirc

Reply:
I'd name the other one `JUNGLE_TREE_1_CONFIG`.

Reply:
it's the sapling tree. renamed